spec/acceptance/place_order_spec.rb

```diff
@@ -91,4 +91,50 @@
   end
 end
 
+describe 'add items' do
+  let(:items_gateway) { FileItemsGateway.new }
+  let(:save_items_details) { SaveItemsDetails.new(items_gateway: items_gateway) }
 
+  context 'given valid item detais' do
```

Comment:
:eyes: If I was to be picky - These tests could pass for a constant implementation, no triangulation seems to be happening